### DISCUSSION TITLE
Any ideas on why ext-ds stub might not be loaded?

### DISCUSSION THREAD

**Discussion by rzvc** [vimeo/psalm#8368] [Q&A] [answered]
Psalm is complaining about Ds\Set having too many template arguments (one), which leads me to believe ext-ds stub is not loaded.

Any ideas on what could cause this?

**Comment by AndrolGenhald** [comment, marked as answer]
If you're using Psalm 5 then you need to require `"ext-ds"` in composer.json or add it with the [`<enableExtensions>`](https://psalm.dev/docs/running_psalm/configuration/#enableextensions) config. I've been wanting to finish up some of the extension changes I had planned and better document it, but time has been hard to find lately.

**Reply by rzvc** [reply]
Thanks!
